### ISSUE TITLE
WIll it work on python3.7.9?

### ISSUE THREAD

**Issue by danielXr** [apple/tensorflow_macos#173]
Need to work with this version....

**Comment by tahuang1991** [comment]
i do not think so.  it detects the python version and exits once version is not python3.8

could be any python3.8, like python3.8.0, 3.8.2, 3.8.8
